stream-chat-android-state/src/main/java/io/getstream/chat/android/state/plugin/factory/StreamStatePluginFactory.kt

```diff
@@ -87,20 +88,20 @@ public class StreamStatePluginFactory(
         val chatClient = ChatClient.instance()
         val repositoryFacade = chatClient.repositoryFacade
         val clientState = chatClient.clientState
-        val globalState = GlobalMutableState.get(chatClient.clientState).apply {
-            clearState()
-            setUser(user)
-        }
+        MutableGlobalStateInstance.clearState()
```

Comment:
Don't we need to set the user?

Reply:
We are [setting the user](https://github.com/GetStream/stream-chat-android/pull/4729/files#diff-d666faec033f7b00886496a4d4a6776e333ac119bdee0eade6120edc5b38df1aR112) after the plugin is created.
It helps us to know that our plugins are ready to be used